GCRCatalogs/utils.py

```diff
@@ -32,3 +33,16 @@ def first(iterable, default=None):
     returns the first element of `iterable`
     """
     return next(iter(iterable), default)
+
+
+def decode(bytestring):
+    """
+    Decode a bytestring is possible, return str if not.
```

Comment:
New docstring is nice and clear, except I believe "is" in line 40 should be "if"

Reply:
Ah, thanks for catching this!